Update readme about vs (#2665)
Azure/azure-sdk-tools: tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md

In the future, passing in a custom cert via a bound volume that contains your certificate will be a possibility as well.

For additional reading on this process for trusting SSL certs locally, feel free to read up [here.](https://devblogs.microsoft.com/aspnet/configuring-https-in-asp-net-core-across-different-platforms/) The afore-mentioned [docker specific readme](../docker/README.md) also has details that are relevant.

## Troubleshooting

### Visual studio

> If you get the message dialog `The project doesn't know how to rin the profile Azure.Sdk.Tools.TestProxy`, you can fix it by reviewing the next two things:

Reviewer: ```suggestion
If you get the message dialog `The project doesn't know how to run the profile Azure.Sdk.Tools.TestProxy`, you can fix it by reviewing the next two things:
```

Also, are these next two things the below ASP.NET and Windows IIS headers? If so I think they should be a lower level of nesting than the current one (`####` vs. `###`).

> ### ASP.NET and web development

Author: ```suggestion
#### ASP.NET and web development
```

Run Visual Studio installer and make sure ASP.NET and web development is installed.

![image](https://user-images.githubusercontent.com/24213737/152257876-be1ed946-20bc-47ff-83da-f9ae05db290a.png)

> Then, look in the right panel that `Development tine IIS support` is not checked:

Reviewer: ```suggestion
Then, confirm in the right panel that `Development time IIS support` is not checked:
```

![image](https://user-images.githubusercontent.com/24213737/152257948-c61e6876-eb36-4414-b8de-8c85aa0532bb.png)

> ### Windows IIS

Author: ```suggestion
#### Windows IIS
```

> [Add Internet Information](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/development-time-iis-support?view=aspnetcore-6.0) Services to your Windows instalation. Here the list of features to enable:

Reviewer: ```
[Add Internet Information](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/development-time-iis-support?view=aspnetcore-6.0) Services to your Windows installation. Here is the list of features to enable:
```
